### PR TITLE
allow_failures: - php: 7.4snapshot removed

### PR DESCRIPTION
**Pull request by WilChrist**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | #861 

#### Summary

I just remove these two lines ```  allow_failures:
    - php: 7.4snapshot``` + the jobs one also (because i think it's not usefull anymore for now)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,10 +22,6 @@ install:
 script:
   - ./vendor/bin/phpunit
 
-jobs:
```
Comment (alcaeus) [inline .travis.yml]: The `jobs` line is still necessary as the resulting YAML file will be invalid otherwise.

Comment (WilChrist) [reply .travis.yml]: Ok, i'll get it back

File: .travis.yml [hunk 2]
```diff
@@ -19,13 +19,11 @@ install:
   - rm composer.lock
   - travis_retry composer update --prefer-dist
 
+jobs:
```
Comment (stof) [inline .travis.yml]: adding it here is wrong. `include` was a child of `jobs` (sibling of `allow_failures`). Moving the top-level key withut moving its content makes no sense (`include` is not nested inside `script`, in an invalid way)

Comment (WilChrist) [reply .travis.yml]: Yes you're right i missed that

**Comment by WilChrist** [comment]
@stof and now ?

**Comment by WilChrist** [comment]
@carusogabriel Thanks for guidance. I'm pretty new in contributing to open source, even in using doctrine, i used it just something like one or two times in projets... and i never look at its internal untill now.

**Comment by alcaeus** [comment]
Superseded by #869. Thanks for your work @WilChrist!

**Pull request closed without merging**